ruby_programming/basic_ruby/project_advanced_building_blocks.md

```diff
@@ -36,7 +36,11 @@ Submit a link below to this [file](https://github.com/TheOdinProject/ruby_course
 
 
 * Add your solution below this line!
+<<<<<<< HEAD
+* [Javier Machin's Solution](https://github.com/Javier-Machin/Ruby_exercises/blob/master/bubble_sort.rb)
+=======
 * [Samuel Masters' Soultion](https://github.com/redeyetuning/rb_adv_bldg_blks/blob/master/bubble_sort.rb)
+>>>>>>> 4951b07eec258113a037f9da01ebd616a1ae663b
```

Comment:
Please delete these merge lines and incorporate both yours and Samuel's solutions into this file